.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,33 @@
+{
+    "configurations": [
+        {
+            "browse": {
+                "databaseFilename": "",
+                "limitSymbolsToIncludedHeaders": true
+            },
+            "includePath": [
+                "/home/braedon/mavros_ws/devel/include/**",
+                "/home/braedon/Catkin/devel/include/**",
+                "/opt/ros/melodic/include/**",
+                "/home/braedon/mavros_ws/src/mavros/libmavconn/include/**",
+                "/home/braedon/Catkin/src/marvelmind_nav/include/**",
+                "/home/braedon/mavros_ws/src/mavros/mavros/include/**",
+                "/home/braedon/mavros_ws/src/mavros/mavros_msgs/include/**",
+                "/home/braedon/Catkin/src/ouster_lib/ouster_client/include/**",
+                "/home/braedon/Catkin/src/ouster_lib/ouster_ros/include/**",
+                "/home/braedon/Catkin/src/ouster_lib/ouster_viz/include/**",
+                "/home/braedon/Catkin/src/pixelink_camera_driver/include/**",
+                "/home/braedon/Catkin/src/robot_localization/include/**",
+                "/home/braedon/Catkin/src/stream_ctl/include/**",
+                "/home/braedon/mavros_ws/src/mavros/test_mavros/include/**",
+                "/usr/include/**"
+            ],
+            "name": "ROS",
+            "intelliSenseMode": "gcc-x64",
+            "compilerPath": "/usr/bin/gcc",
+            "cStandard": "c11",
+            "cppStandard": "c++17"
+        }
+    ],
+    "version": 4
+}
```

Comment:
Please do not include editor configs that specific to your computer.

Reply:
Note: you can add `.vscode` to `.gitignore`.

Reply:
Darn Ive been so careful about not staging that but was going to ask about the .gitignore.